llvm/utils/repo/Makefile

```diff
@@ -8,7 +8,12 @@
 
 # The path of the directory in which llvm-project-prepo sources is checked out from git.
 LLVM_REPO = ${HOME}/llvm-project-prepo
-TOOLCHAIN_FILE=$(LLVM_REPO)/llvm/utils/repo/repo.cmake
+# Choose the toolchina file to be used. By default, the repo.cmake is used.
+ifeq ($(FULL_REPO),yes)

```

Comment:
No sure about testing for ths explicit string "yes". Use ifdef instead?

Reply:
Fixed in the commit https://github.com/SNSystems/llvm-project-prepo/pull/155/commits/ce1d66d51a4d69b1046d9f2abdb013408110f63b.